internal/lock/sync.go

```diff
@@ -0,0 +1,92 @@
+package lock
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	"go.jetpack.io/devbox/internal/cuecfg"
+)
+
+func SyncLockfiles() error {
+	latestPackages, err := latestPackages()
+	if err != nil {
+		return err
+	}
+
+	// Step 2: Update the devbox.lock files
+	return filepath.Walk(".", func(path string, fi os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+
+		if !fi.IsDir() && filepath.Base(path) == "devbox.lock" {
```

Comment:
Is this going to walk all of my `node_modules` files, or are we excluding directories according to `.gitignore` somewhere

Reply:
That's a good call. Yeah I think using `.gitignore` is good.

Reply:
Want to add that the monorepo might have a top level `.gitignore` that handles all the sub directories, and individual `.gitignore` in each subdirectory to add more stuff to ignore